httpcore/_backends/trio.py

```diff
@@ -143,7 +143,10 @@ async def open_tcp_stream(
         port: int,
         ssl_context: Optional[SSLContext],
         timeout: TimeoutDict,
+        local_addr: Optional[bytes],
     ) -> AsyncSocketStream:
+        if local_addr:
```

Comment:
Maybe a comment that it is currently supported in `trio` master, and should be expected from version 0.16.1 onwards?

Also, what will our implementation change look like once trio support does land here?

Reply:
I'm adding a comment.

I believe the code would just add something like `local_address=local_addr` to the `open_tcp_stream` call, although it's possible that there would be a type mismatch, as the `local_addr` parameter is `None` or a `bytes`, and trio claims to want `None` or a `str` (although it just passes it into `socket.bind`, which should accept a `bytes`).